_posts/2012-03-05-source-code-to-apprtc.appspot.com-example-app-available.md

```diff
@@ -7,7 +7,7 @@ date: 2012-03-05 14:21:00
 
 
 The source code to <https://apprtc.appspot.com/> is now available at
-<https://chromium.googlesource.com/external/webrtc/+/master/talk/examples/android/src/org/appspot/apprtc/>
+https://chromium.googlesource.com/external/webrtc/+/master/talk/examples/android/src/org/appspot/apprtc/ (link outdated).
```

Comment:
Actually, that's probably not right. This is the one: https://chromium.googlesource.com/external/webrtc/+/master/webrtc/examples/androidapp/src/org/appspot/apprtc